pkg/ccl/backupccl/backup_planning.go

```diff
@@ -445,25 +455,32 @@ func backupPlanHook(
 				if err != nil {
 					return errors.Wrapf(err, "determining base for incremental backup")
 				}
-				prevBackups = make([]BackupManifest, 0, len(prev)+1)
+				prevBackups = make([]BackupManifest, len(prev)+1)
 
 				m, err := readBackupManifestFromStore(ctx, defaultStore, encryption)
 				if err != nil {
 					return errors.Wrap(err, "loading base backup manifest")
 				}
-				prevBackups = append(prevBackups, m)
+				prevBackups[0] = m
 
 				if m.DescriptorCoverage == tree.AllDescriptors &&
 					backupStmt.DescriptorCoverage != tree.AllDescriptors {
 					return errors.Errorf("cannot append a backup of specific tables or databases to a full-cluster backup")
 				}
 
-				for _, inc := range prev {
-					m, err := readBackupManifest(ctx, defaultStore, inc, encryption)
-					if err != nil {
-						return errors.Wrapf(err, "loading prior backup part manifest %q", inc)
-					}
-					prevBackups = append(prevBackups, m)
+				for i, inc := range prev {
```

Comment:
ditto here -- either shadow `inc` too or, imo, better yet don't bind it.

Reply:
Ah, updated the usage in readBackupManifest but missed the other usage. I think my eyes are at their limit for the day.